Client/src/Components/Charts/CustomGauge/index.jsx

```diff
@@ -29,6 +29,15 @@ const MAXIMUM_VALUE = 100;
  */
 const CustomGauge = ({ progress = 0, radius = 70, strokeWidth = 15, threshold = 50 }) => {
 	const theme = useTheme();
+	const BOX_STYLE = {
+		borderRadius: "50%",
+		boxShadow: "0px 2px 4px -3px #20202040",
+		padding:"6px",
+		alignItems:"center",
+		display:"flex",
+		backgroundColor:theme.palette.background.main
+	};
```

Comment:
Styling must be consistent throughout the app, the theme should be used to apply styles to something like this